lib/jobs.rb

```diff
@@ -28,7 +29,7 @@ def self.establish_connection!
   end
 
   def self.scrub_query(query)
-    query.remove(/\b(position|opening|posting|job|employment)s?\b/, '').squish
+    query.remove(/\b#{SCRUB_KEYWORDS}\b/i, '').squish
```

Comment:
I like this better - much cleaner.

You only need to pass the regex to `remove` (sorry I didn't catch that last time):
```ruby
query.remove(/\b#{SCRUB_KEYWORDS}\b/i).squish
```